packages/forms/src/directives/validators.ts

```diff
@@ -593,6 +584,8 @@ export class PatternValidator implements Validator, OnChanges {
   }
 
   private _createValidator(): void {
-    this._validator = Validators.pattern(this.pattern);
+    if (this.pattern) {
```

Comment:
Can `""` be a valid pattern? Perhaps you should be doing this instead here?

```suggestion
    if (this.pattern !== undefined) {
```

Reply:
test is failing.
it seems that I need to add a test for this situation ?